src/orders/delivery/app.py

```diff
@@ -0,0 +1,110 @@
+# Python's libraries
+from typing import Dict
+from typing import Any
+import re
+
+# import json
+
+# Own's modules
+from location_router import TravelPlanner
+from delivery_modules.dao.order_dao import OrderDAO
+from delivery_modules.processors.order_helpers import OrderProcessor
+from delivery_modules.utils.doorman import DoormanUtil
+from delivery_modules.errors.auth_error import AuthError
+
+from settings import ORDERS_PRIMARY_KEY
+
+# Third-party libraries
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+
+def set_delivery_schedule_order(
+    event: Dict[str, Any], context: LambdaContext
+) -> Dict[str, Any]:
+    """This function will retrieve todays orders to generate an optimized path.
+
+    :param event: Custom object that can come from an APIGateway, nothing relevant in the body.
+    :type event: Dict
+    :param context: Regular lambda function context
+    :type context: LambdaContext
+    :return: Custom object with the reponse from the lambda, it could be a 200, if the resources were found
+    or >= 400 if theras was an error
+    :rtype: Dict
+    """
+
+    logger = Logger()
+    logger.info("Initializing set_delivery_schedule_order function")
+    try:
+        doorman = DoormanUtil(event, logger)
+        username = doorman.get_username_from_token()
+        is_auth = doorman.auth_user()
+        if is_auth is False:
+            raise AuthError("User is not allow to retrieve orders")
+        body = doorman.get_body_from_request()
+        schedule_for_date = body["date"]
+        pattern = re.compile(r"^\d{4}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])$")
+        logger.info(f"Date to process: {schedule_for_date}")
+        if bool(pattern.match(schedule_for_date)) is False:
```

Comment:
I would use datetime instead of regex: 
```
    try:
        datetime.strptime(schedule_for_date, '%Y-%m-%d')
    except ValueError:
        raise ValueError("Date format is not valid")
```